src/relentless/simulate/simulate.py

```diff
@@ -566,12 +595,18 @@ class PairPotentialTabulator(PotentialTabulator):
         potential.
     neighbor_buffer : float
         Buffer radius used in computing the neighbor list.
+    exclusions : list
+        The neighborlist nominally includes all pairs within ``rmax`` of each other.
+        This option allows for exclusions of pairs that should not be included in the
+        neighbor list. The string should be formatted as a tuple of strings, e.g.,
+        ``['bond', '1-3', 1-4]``.
```

Comment:
Here again, I think we said 1-2 right? I would also add a bulleted list of allowed values in this documentation and what they mean.